### PR TITLE
Bump bolt to v1.1.0

### PR DESCRIPTION
**Pull request by hqhq**
It adds ARM64, ppc64le, s390x, solaris support, and a bunch of
bugfixs.

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

**Comment by hqhq** [comment]
Not sure why the test failed, seems unrelated.

File: hack/vendor.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ clone git github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc37
 clone git github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
 clone git github.com/coreos/go-etcd v2.0.0
 clone git github.com/hashicorp/consul v0.5.2
-clone git github.com/boltdb/bolt v1.0
+clone git github.com/boltdb/bolt 14ea02b7ee56ea0440b8e42b333934fdf623f214
```
Comment (jfrazelle) [inline hack/vendor.sh]: can we ask them to make a tag instead, small detail but no worries if not

Comment (tianon) [inline hack/vendor.sh]: Yes please!  https://github.com/boltdb/bolt/issues/438

**Comment by brahmaroutu** [comment]
@hqhq Can you please bump up the revision to 1.1 for bolt. It is revision now.

**Comment by hqhq** [comment]
@brahmaroutu Updated.
@jfrazelle @tianon please review again, thanks~

**Comment by calavera** [comment]
LGTM

**Comment by duglin** [comment]
LGTM but testing locally to make sure all is well.

**Comment by duglin** [comment]
testing locally worked so merging!

**Pull request merged**